src/components/v5/frame/ColonyHome/partials/ReputationChart/partials/DropdownMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import { DotsThreeVertical } from '@phosphor-icons/react';
+import clsx from 'clsx';
+import React, { type FC, type PropsWithChildren } from 'react';
+
+interface DropdownMenuProps extends PropsWithChildren {
+  isDisabled?: boolean;
+  setTriggerRef: (node: HTMLElement | null) => void;
+  setTooltipRef: (node: HTMLElement | null) => void;
+  getTooltipProps: () => Record<string, any>;
+  visible: boolean;
+}
+
+const DropdownMenu: FC<DropdownMenuProps> = ({
```

Comment:
should we maybe already extract this to a new shared component? maybe we can reuse this for the team filter too :eyes: